### DISCUSSION TITLE
如何将符号表分节？

### DISCUSSION THREAD

**Discussion by GorgeShao** [obster-y/XJTU-thesis#97] [Q&A]
我想制作如下格式的分节的符号表，请问需要哪些代码呢？大佬能帮忙给一个简单的代码示例吗？

==== 以下是分节的符号表的例子 ====
主要符号表

A. 上下标与索引
     i     机组编号
    m   线路编号

B. 决策变量
    p    有功功率潮流
    q    无功功率潮流

**Comment by sikouhjw** [comment]
看起来你需要的是 https://tex.stackexchange.com/a/530586

**Reply by GorgeShao** [reply]
很感谢您的回答，但是这个解决方案貌似不适用于交大博士论文的Latex模板

**Reply by sikouhjw** [reply]
> 很感谢您的回答，但是这个解决方案貌似不适用于交大博士论文的Latex模板

你既想要特殊的效果，又不愿意打破现有的框架，那能怎么办呢

**Reply by obster-y** [reply]
没有什么方案，因为学校没有这样的要求。而且，这像是 nomenclature 而不是 glossary。这个不是一些期刊，所以按照学校要求来比较好，建议不要自创规则。如果非要要，可以自己 patch 或者重新写一个 glossary
